__tests__/version/get-version.test.ts

```diff
@@ -1,68 +1,122 @@
-import {mockOldestQueryResponse} from './graphql.mock'
+import {rest} from 'msw'
+import {setupServer} from 'msw/node'
 import {
   getOldestVersions as _getOldestVersions,
-  QueryInfo
+  RestQueryInfo
 } from '../../src/version'
 import {Observable} from 'rxjs'
+import {getMockedVersionsResponse} from './rest.mock'
 
-describe.skip('get versions tests -- call graphql', () => {
-  it('getOldestVersions -- succeeds', done => {
-    const numVersions = 1
-    getOldestVersions({numVersions}).subscribe(result => {
-      expect(result.versions.length).toBe(numVersions)
-      done()
-    })
+const RATE_LIMIT = 99
```

Comment:
The RATE_LIMIT was a limitation due to limitation of graphql api. Do we have the same limitation with REST or can it be updated/removed?

Reply:
The rate limit is 1000 requests per repository per hour when using GITHUB_TOKEN and 5000 requests per user per hour when using PAT. Considering the limit with GITHUB_TOKEN, I think 100 is sufficient as we would not want this workflow to exhaust a significant part of limit and possibly cause issue with other workflows.

Reply:
Makes sense. 👍